client/src/ui/molecules/mandala/index.tsx

```diff
@@ -203,6 +223,15 @@ function Mandala({
                 <tspan>&lt;&gt;</tspan>&lt;/&gt;
               </textPath>
             </text>
+            {pride && (
+              <text dy="70" textLength="750">
+                <textPath textLength="755" href="#circle6">
```

Comment:
```suggestion
              <text dy="70" textLength="754">
                <textPath textLength="754" href="#circle6">
```

that's `2 * 120 * 𝜋 ≅ 754`